runtime/doc/builtin.txt

```diff
@@ -3786,6 +3787,14 @@ getbufvar({buf}, {varname} [, {def}])				*getbufvar()*
 		Return type: any, depending on {varname}
 
 
+getcellpixels()						*getcellpixels()*
+		Returns a |List| of cell pixel size.
+		List format is [xpixels, ypixels].
+		Only on UNIX.
+
+		Return type: list<any>
```

Comment:
The document should explain that some environments may return unreliable values.

Reply:
On what environments would it be unreliable? I think it just needs to explain that it would return an error value, e.g. -1,-1 when it's not available. That's not unreliable.